src/Tracing.cpp

```diff
@@ -72,6 +72,18 @@ class InjectTracing : public IRMutator2 {
             trace_parent = Variable::make(Int(32), op->name + ".trace_id");
         } else if (op->call_type == Call::Image) {
             trace_it = trace_all_loads;
+            // If there is a Function in the env named "name_im", assume that
```

Comment:
name_im could be scheduled and have actual loads from it (via ImageParam::in). I think we should only trace loads from the ImageParam if name_im is scheduled inline.

Reply:
Fixed